src/pages/kyubinkim/Login/Login.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import "../Login/Login.scss";
+
+class LoginKyubin extends Component {
+  state = {
+    userName: "",
+    userPass: "",
+    infoChecked: false,
+    passChecked: false,
+    buttonColor: "#bedffc",
+    move: false,
+  };
+
+  // Main으로 가기
+  goToMain = () => {
+    if (this.state.move) {
+      this.props.history.push("/Main-kyubin");
+    }
+  };
+
+  //input @ 체크
+  infoInputCheck = (e) => {
+    this.setState({ userName: e.target.value });
+    if (e.target.value.includes("@")) {
+      this.setState({ infoChecked: true }, () => this.color());
+    } else {
+      this.setState({ infoChecked: false }, () => this.color());
+    }
+  };
+  //pass 5자 이상 체크
+  passInputCheck = (e) => {
+    this.setState({ userPass: e.target.value });
+    if (e.target.value.length >= 5) {
+      this.setState({ passChecked: true }, () => this.color());
+    } else {
+      this.setState({ passChecked: false }, () => this.color());
+    }
+    console.log(e.target.value);
+  };
+  //box 컬러 변경
+  color = () => {
+    if (this.state.infoChecked && this.state.passChecked) {
+      this.setState({ buttonColor: "#1296f7" });
+      this.setState({ move: true });
+    } else {
+      this.setState({ buttonColor: "#bedffc" });
+      this.setState({ move: false });
+    }
+  };
```

Comment:
```suggestion
  handleBtnActivation = () => {
    const { infoChecked, passChecked } = this.state;
    const isBtnActive =  infoChecked && passChecked;
    this.setState({ isBtnActive })
  };
```
- 함수 이름에는 기능이 드러나게 해주세요. 나중에 함수형 컴포넌트 사용하려면 더욱 중요합니다.
- 비구조화할당을 생활화합시다. 자바스크립트 ES6 문법입니다. 리팩토링 체크리스트 확인해주세요.
- 하나의 조건으로 두 가지 데이터를 관리하기 보다는 하나로 합쳐줄 수 있겠네요.
- 해당 데이터에 따라서 goToMain 함수 실행 여부도, 버튼 색도 관리할 수 있겠네요.
- 버튼 색을 직접 state에서 관리하지 않고 삼항연산자를 사용해 className을 관리해줍니다.
- 다시 한 번 말씀드리지만 함수형 컴포넌트는 2차 프로젝트 부터 적용해주세요.

Reply:
와 직접 함수까지  ..! 감사합니다 수정완료했습니다.